data/src/main/java/pl/edu/agh/model/Project.java

```diff
@@ -1,5 +1,36 @@
 package pl.edu.agh.model;
 
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Transient;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
 public class Project {
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(length = 64)
+    private String name;
+    @Transient //TODO add missing mapping
+    private List<User> usersInProject;
+    @Transient //TODO add missing mapping
+    private List<Sprint> sprints;
+    @Column
+    private String description;
+    @Column
+    private double startingCoefficient;
+
```

Comment:
ktoś wcześniej płakał nad tym enterem
i ładniej jakby pomiędzy każdym polem klasy był odstęp

Reply:
(To było zresolvowane, ale nadal jest)

Reply:
Ok, ja wiem, że wszyscy wiemy, że czepianie się pustych linijek jest pedantyzmem, ale...
Plus nie zaszkodziłoby `Ctrl + Alt + L`, bo brakuje spacji przed klamerką

Reply:
I w sumie w innych klasach nie mamy też pustych linijek na początku klasy. KONSEKWENCJA JEST FAJNA

Reply:
Poprawiłem pozostałe klasy zgodnie z konwencją